CONTRIBUTING.md

```diff
@@ -71,7 +71,7 @@ breaking         -> Runs the breaking changes checker against a package
 
 Basic usage of `tox` within this monorepo is:
 
-1. `pip install tox<5`
+1. `pip install 'tox<5'`
```

Comment:
@FacerAin thanks for your contribution! I tested this with powershell, command prompt, and WSL and it seems to me that we need double quotes around tox, like `"tox<5"`. Would you mind updating the PR?

Reply:
I fix it with the **double quotes** you mentioned. 
Thank you for the review ☺️